src/column/AColumn.ts

```diff
@@ -116,34 +116,53 @@ abstract class AColumn<T, DATATYPE extends IDataType> extends EventHandler {
       .html(`
         <aside></aside>
         <header class="columnHeader">
-          <div class="labelName"><i class="${dataValueTypeCSSClass(dataValueType(this.data))}" aria-hidden="true"></i> <span>${formatAttributeName(this.data.desc.name)}</span></div>
-          <div class="toolbar"></div>
+          <div class="toolbar">
+            <div class="labelName"><i class="${dataValueTypeCSSClass(dataValueType(this.data))}" aria-hidden="true"></i> <span>${formatAttributeName(this.data.desc.name)}</span></div>
+            <div class="onHoverToolbar"></div>
+          </div>
         </header>
         <main></main>`);
 
     this.buildToolbar($node.select('div.toolbar'));
-
+    $node.select('div.onHoverToolbar')
+      .style('display', 'none' )
+      .style('visibility', 'hidden');
+    $node.select('header').on('mouseover', () => {
+      $node.select('div.onHoverToolbar')
+        .style('top', '31px')
```

Comment:
Why do you define static values here in TypeScript? These kind of values belong with a CSS class into the CSS files.